django/contrib/admin/static/admin/css/base.css

```diff
@@ -44,14 +44,14 @@ html[data-theme="light"],
     --button-fg: #fff;
     --button-bg: var(--primary);
     --button-hover-bg: #609ab6;
-    --default-button-bg: var(--secondary);
+    --default-button-bg: var(--button-fg);
```

Comment:
This effectively makes primary buttons harder to spot than secondary buttons so isn’t the right way forward.